includes/admin/settings/register-settings.php

```diff
@@ -892,7 +892,7 @@ function edd_get_registered_settings() {
 						'size' => 'regular',
 					),
 					'show_privacy_policy_on_checkout' => array(
-						'id'   => 'show_agree_to_privacy_policy_on_checkout',
+						'id'   => 'show_privacy_policy_on_checkout',
```

Comment:
We need an upgrade routine here. At least 7,000 sites have already updated so we have to ensure that those who have enabled this do not lose their setting.

Reply:
@pippinsplugins ok we should just be able to do a quick swap that auto runs, no need for them to click anything.

Reply:
@pippinsplugins fixed.